console-subscriber/src/lib.rs

```diff
@@ -163,6 +163,12 @@ struct Shared {
     /// flushed.
     flush: aggregator::Flush,
 
+    /// As fill percentage gets higher, aggregator task gets polled more often,
+    /// to avoid filling up to the flush point. While `flush` notifiew the
```

Comment:
a couple minor docs nits:
```suggestion
    /// As fill percentage gets higher, the aggregator task gets woken more often,
    /// to avoid filling up to the flush point. While `flush` notifies the
```